lfpykernels/kernel_approximator.py

```diff
@@ -481,11 +496,15 @@ def get_kernel(self, probes, Vrest=-65, dt=2**-4,
                 # NOTE: ignoring shifting synapse placements by the mean
                 # somatic depth, which may be implemented as:
                 # syn_pos['funargs'][h]['loc'] = \
-                #     funarg['loc'] + self.populationParameters['loc']
+                #    funarg['loc'] + self.populationParameters['loc']
+                # TODO: Any reason why we can't just implement
+                # this as a standard
```

Comment:
I think doing it this way would be a bit wrong. Even if the somas in a certain layer have a wide distribution, one could assume that they are synaptically targeted quite narrowly within a certain layer, e.g, L1. Convolving these distributions with each other would not allow for that.

Reply:
As for the type of distributions, we can perhaps generalize this to any type of distribution in scipy.stats, but it would mean some more meddling with parsing the corresponding parameters (not always `loc` and `scale`).

Reply:
I don't fully understand: Do you mean that implementing this as a standard convolution would be wrong? For two gaussian distributions, would this not give exactly the same result as is here now?